test/docker/compose.go

```diff
@@ -522,6 +522,7 @@ func (d *Compose) startCluster(ctx context.Context, size int, settings map[strin
 	settings["RAFT_INTERNAL_RPC_PORT"] = "8301"
 	settings["RAFT_JOIN"] = raft_join
 	settings["RAFT_BOOTSTRAP_EXPECT"] = strconv.Itoa(d.size)
```

Comment:
It would be helpful to have a separate acceptance test for the recovery mechanism

Reply:
isn't `TestActivationDeactivation` does the job, or what you have in mind ?

Reply:
the ip changes for all the containers which they have stop/start in place, but I can add more test case sure

Reply:
That would be nice

Reply:
done